travis-scripts/script.sh

```diff
@@ -19,9 +19,7 @@ INSTDIR=$HOME/cf_install
 # fi
 
 # Unshallow the clone. Fetch the tags from upstream even if we are on a
```

Comment:
I wonder if we still need this. determine-version.py needed all tags to work, but new determine-version.sh needs only to check if the tip of the current branch points to a tag.

Reply:
Yes, I was thinking the same. I can test later.